config/helm/schema.yaml

```diff
@@ -125,23 +191,151 @@ properties:
     type: string
-    description: Make sure you have applied the latest CRD by following the instructions in the Github Repository.
-    title: CRD manually applied
-    required: true
+    title: Webhook Memory request
+    description: |
+      The amount of memory allocation the Webhook requests from the cluster.
+      It is recommended for the memory requests to be the same as the memory limits.
+    default: 128Mi
+  webhook.limits.cpu:
+    type: string
+    title: Webhook CPU limit
+    description: |
+      The amount of CPU allocation to which the cluster may limit the Webhook.
+      It is recommended for the CPU limits to be the same as the CPU requests.
+    default: 300m
+  webhook.limits.memory:
+    type: string
+    title: Webhook Memory limit
+    description: | 
+      The amount of memory allocation to which the cluster may limit the Webhook.
+      It is recommended for the memory limits to be the same as the memory requests.
+    default: 128Mi
+  webhook.highAvailability:
+    type: boolean
+    title: Enables Webhook's high availability mode
+    description: Increases replicas and adds topology constraints for the Webhook.
+    default: false
+
+  csidriver.enabled:
+    type: boolean
+    title: Enables CSI driver
+    description: The CloudNativeFullStack as well as AppOnly mode need the CSI driver to function properly.
+    default: false
+  csidriver.requests.cpu:
+    type: string
+    title: CSI driver CPU request
+    description: |
+      The amount of CPU allocation the CSI driver requests from the cluster.
+      It is recommended for the CPU requests to be the same as the CPU limits.
+    default: 300m
+  csidriver.requests.memory:
+    type: string
+    title: CSI driver Memory request
+    description: |
+      The amount of memory allocation the CSI driver requests from the cluster.
+      It is recommended for the memory requests to be the same as the memory limits.
+    default: 100Mi
+  csidriver.limits.cpu:
+    type: string
+    title: CSI driver CPU limit
+    description: |
+      The amount of CPU allocation to which the cluster may limit the CSI driver.
+      It is recommended for the CPU limits to be the same as the CPU requests.
+    default: 300m
+  csidriver.limits.memory:
+    type: string
+    title: CSI driver Memory limit
+    description: |
+      The amount of memory allocation to which the cluster may limit the CSI driver.
+      It is recommended for the memory limits to be the same as the memory requests.
+    default: 100Mi
+
+  activeGate.readOnlyFs:
+    type: boolean
+    title: Mount filesystem in read-only mode
+    description: Restricts ActiveGate deployments to mount filesystems read-only.
+    default: false
+
+  installCRD:
+    type: boolean
+    title: Install DynaKube CRD
+    description: |
+      Tells the Helm chart of the deployer to install the custom resource definition (CRD) of the DynaKube API.
+    default: true
     enum:
-    - 'yes'
+      - true
```

Comment:
does the enum needed ?
with this user can't set `installCRD` to `false`

Reply:
The schema is used for inital installation of the operator. Therefore it makes no sense to not install it

Reply:
@chrismuellner should we give the option ? (I think we should)